src/main/java/org/icgc/argo/workflow_management/service/NextflowService.java

```diff
@@ -256,7 +261,7 @@ private CmdKubeRun createCmd(@NonNull Launcher launcher, @NonNull WESRunParams p
     cmdParams.put("runConfig", List.of(config));
 
     // Resume workflow by name/id
-    cmdParams.put("resume", workflowEngineOptions.getResume());
+    cmdParams.put("resume", workflowEngineOptions.getResume().toString());
```

Comment:
just a final comment, your local variable for WorkflowEngineParams is called workflowEngineOptions

Reply:
workflowEngineOptions

> BadRequest

BadRequest